### DISCUSSION TITLE
stable diffusion desktop app has entered the preview version

### DISCUSSION THREAD

**Discussion by Cyberhan123** [leejet/stable-diffusion.cpp#126] [Show and tell]
Now that the basic functions of the stable diffusion desktop have been developed.

<img width="1039" alt="截屏2024-01-11 00 09 03" src="https://github.com/leejet/stable-diffusion.cpp/assets/38837039/f3c599c8-3652-41b6-86b5-530ea309ed5f">

project address: https://github.com/Cyberhan123/stable-diffusion-desktop
download link: https://github.com/Cyberhan123/stable-diffusion-desktop/releases/latest
known defect: https://github.com/Cyberhan123/stable-diffusion-desktop/issues

**Comment by Cyberhan123** [comment]
@FSSRepo @leejet @Green-Sky

**Comment by Amin456789** [comment]
nice! cant wait for a windows version and a stable release in the future with all of the features, thanx

**Comment by Amin456789** [comment]
@FSSRepo @Cyberhan123 any more news of the windows version? im so excited about this gui, will be great if u guys add select lcm lora and taesd option too

**Reply by Cyberhan123** [reply]
I'm working on the pre-process, you can see my pr.

**Reply by Amin456789** [reply]
cant wait for its release, tnx

**Reply by Cyberhan123** [reply]
The basics are now done, but there's still a lot of work to be done, I mean it's just a toy

**Reply by Amin456789** [reply]
this is so great, thank u so much, @leejet should mention this in the read me so people can use it a lot. can't wait for the support of lcm lora and taesd [in my laptop without these two it takes a lot of time to make images], i see u r working on image 2 image and random seed too, cant wait for them, also could u please add a dark mode too, in the night its easier with dark mode
overall, this is amazing, thanks again for ur hard work

**Comment by Amin456789** [comment]
woohoo! new updates are so cool! we now can use sd cpp without using cmd at all!
